src/Components/CreateImageWizard/steps/Registration/index.tsx

```diff
@@ -76,6 +79,9 @@ const RegistrationStep = () => {
             <ActivationKeyInformation />
           </FormGroup>
         )}
+      {process.env.IS_ON_PREMISE && registrationType !== 'register-later' && (
```

Comment:
There's slightly weird behaviour - when you select "Register later" go to another step and then return to Registration the "additional connection options" checkbox array gets automagically expanded :magic_wand: 

![expand](https://github.com/user-attachments/assets/ef43df43-0c92-4bc8-b480-8c3f1a683d15)